### PR TITLE
Issue #17388: Update MyFaces 3.0.0 to 3.0.1

### LINKED ISSUE #17388: Pull in MyFaces 3.0.1 once available

### PR DESCRIPTION
**Pull request by pnicolucci**
fixes #17388

### CODE REVIEW

**Comment by pnicolucci** [comment]
#build

File: dev/io.openliberty.org.apache.myfaces.3.0/bnd.bnd [hunk 1]
```diff
@@ -119,7 +119,7 @@ Import-Package: \
   *
 
 -buildpath: \
-  org.apache.myfaces.core:myfaces-impl;version=3.0.0,\
+  org.apache.myfaces.core:myfaces-impl;version=3.0.1,\
```
Comment (volosied) [inline dev/io.openliberty.org.apache.myfaces.3.0/bnd.bnd]: Want to switch this over to Implementation-Version?  One less change later on

**Comment by pnicolucci** [comment]
#build

**Review by volosied — APPROVED** [review]
Looks good, thanks!

**Review by wtlucy — APPROVED** [review]

**Comment by pnicolucci** [comment]
#run-libby-bot

**Comment by pnicolucci** [comment]
#build #spawn.fullfat.buckets=com.ibm.ws.jsf.2.3_fat,com.ibm.ws.jsf.2.2_fat

**Pull request merged**